### PR TITLE
add certificate title override

### PR DESCRIPTION
**Pull request by zubair-arbi**
ECOM-3557
@awais786 @ahsan-ul-haq @tasawernawaz 
* Show `title` from the program certificate configurations if set, instead of the program name while rendering the certificate
* User site configuration for linking to its lms url
* Add/Update test for above changes

_Old PR: https://github.com/edx/credentials/pull/49_

### CODE REVIEW

**Comment by awais786** [comment]
:+1:

**Comment by zubair-arbi** [comment]
@clintonb Please review.

File: credentials/templates/credentials/program_certificate.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
         <div class="wrapper-header">
             <header class="header-app" role="banner">
                 <h1 class="header-app-title">
-                    <a class="logo" href="http://edx.org">
-                        <img class="logo-img" src="{% static 'images/logo-edX.png' %}" alt="{{ platform_name }} {% trans 'Home' %}" />
+                    <a class="logo" href="{{ site.siteconfiguration.lms_url_root }}">
+                        <img class="logo-img" src="{% static 'images/logo-edX.png' %}" alt="{% firstof site.name platform_name %}" />
```
Comment (clintonb) [inline credentials/templates/credentials/program_certificate.html]: Is there a ticket to properly handle theming?

Comment (clintonb) [inline credentials/templates/credentials/program_certificate.html]: Use `{% firstof site.name platform_name as site_name %}` to create a variable named `site_name` and avoid the duplication below.

Comment (zubair-arbi) [inline credentials/templates/credentials/program_certificate.html]: @clintonb we have a ticket for theming: [ECOM-3595](https://openedx.atlassian.net/browse/ECOM-3595)
The “as” syntax for [`firstof` was added in `Django 1.9`](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#firstof) so we cannot use this syntax here.

**Comment by clintonb** [comment]
:+1:

**Pull request merged**